rollup.config.js

```diff
@@ -6,19 +6,19 @@ export default {
   format: 'umd',
   plugins: [
     babel({
-        babelrc: false,
-        presets: [
-          "es2015-rollup"
-        ],
-        plugins: [
-          [
-            "transform-es2015-classes",
-            {
-              "loose": true
-            }
-          ]
+      babelrc: false,
+      presets: [
+        'es2015-rollup'
+      ],
+      plugins: [
+        [
+          'transform-es2015-classes',
+          {
+            loose: true
+          }
         ]
-      }
+      ]
+    }
```

Comment:
eslint --fix did it for me so I doubt it's wrong :)

if you look carefully, you'll see it's still indented, just by 2 spaces instead of 4.